includes/admin/tools/export/class-batch-export-donors.php

```diff
@@ -83,20 +83,21 @@ public function __construct( $_step = 1 ) {
 	 *
 	 * @since 2.1.0
 	 *
-	 * @param $filename
-	 * @param $export_type
+	 * @param string $filename File name.
+	 * @param string $export_type export type.
 	 *
-	 * @return string
+	 * @return string $filename file name.
 	 */
-	function give_export_filename( $filename, $export_type ) {
-
+	public function give_export_filename( $filename, $export_type ) {
 
 		if ( $this->export_type !== $export_type ) {
 			return $filename;
 		}
 
-		if ( ! empty( (int) $_GET['forms'] ) ) {
-			$slug     = get_post_field( 'post_name', get_post( absint( $_GET['forms'] ) ) );
+		$forms = empty( $_GET['forms'] ) ? false : absint( $_GET['forms'] );
```

Comment:
@raftaar1191 set default value to `0` instead of `false`

Reply:
Done @ravinderk